tests/common/plugins/sanity_check/__init__.py

```diff
@@ -94,10 +94,12 @@ def do_checks(request, check_items, *args, **kwargs):
     for item in check_items:
         check_fixture = request.getfixturevalue(item)
         results = check_fixture(*args, **kwargs)
+        logger.debug("check results of each item {}".format(results))
         if results and isinstance(results, list):
             check_results.extend(results)
         elif results:
             check_results.append(results)
+    logger.debug("check_results {}".format(check_results))
```

Comment:
This is unnecessary. The `check_results` returned by `do_checks` is already logged in line 230 and 250.
Too much redundant information will make it harder to search through the test log to find useful information.

Reply:
I delete this.